### PR TITLE
feat: create seed for onboarding journey

### PR DESCRIPTION
**Pull request by csiyang**
# Description

Adds a onboarding journey for dev use, with the id and slug matching the real onboarding journey in production

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/31822575/todos/5910584578)

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [x] Run `nx seed api-journeys`
- [x] You should have dev onboarding journey in template library that can be used

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged into main

### CODE REVIEW

File: apps/api-journeys/db/seeds/onboarding.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+import {
+  JourneyStatus,
+  ThemeMode,
+  ThemeName
+} from '../../src/app/__generated__/graphql'
+
+const db = ArangoDB()
+
+export async function onboarding(action?: 'reset'): Promise<void> {
+  // reset should only be used for dev and stage, using it on production will overwrite the existing onboarding journey
```
Comment (csiyang) [inline apps/api-journeys/db/seeds/onboarding.ts:12]: Still debating with myself if this feature should even be included

File: apps/api-journeys/db/seeds/onboarding.ts [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  `)
+
+    await db.query(aql`
+    FOR journey in journeys
+      FILTER journey.slug == ${slug}
+      REMOVE journey in journeys
+  `)
+  }
+
+  const existingJourney = await (
```
Comment (csiyang) [inline apps/api-journeys/db/seeds/onboarding.ts:32]: This is so if this seed ever runs on production, it won't overwrite the actual onboarding journey. But it also makes it harder to update the onboarding journey for stage and dev

**Comment by csiyang** [comment]
I will run this seed for stage once this is approved and merged

**Review by storyworks — APPROVED** [review]
Looks good, works with mine too. Can you also make a post in `nextsteps-frontend` to remind people to run seed locally?

**Pull request merged**